src/hotspot/share/utilities/globalCounter.hpp

```diff
@@ -86,4 +92,8 @@ class GlobalCounter : public AllStatic {
   class CriticalSection;
 };
 
+ENUMERATOR_VALUE_RANGE(GlobalCounter::CSContext,
```

Comment:
There is not, so far as I know, any use-case for iteration over CSContext values.  And those two don't define the range of such.  They are not declared as CSContext values, nor should they be.